api/src/test/scala/com/github/andyglow/jsonschema/AsDraft07Spec.scala

```diff
@@ -7,6 +7,8 @@ import json.schema.Version._
 import org.scalatest.matchers.should.Matchers._
 import org.scalatest.wordspec.AnyWordSpec
 
+case class FooX(a: Int)
+case class FooY(b: Int)
```

Comment:
can we have these guys moved into spec's companion object?

Reply:
done